assets/src/data/eventespresso/core/actions/reset.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * External imports
+ */
+import {
+	singularModelName,
+	pluralModelName,
+} from '@eventespresso/model';
+import { some, keys } from 'lodash';
+import { isModelEntityOfModel } from '@eventespresso/validators';
+import { select as dataSelect } from '@wordpress/data';
+
+/**
+ * Internal imports
+ */
+import { dispatch, select } from '../../base-controls';
+import { ACTION_TYPES } from './action-types';
+import { REDUCER_KEY } from '../constants';
+import * as modelSpecificSelectors from '../model/model-selectors-index';
+
+const { resets: types } = ACTION_TYPES;
+
+/**
+ * Resets the entire state to its default for the store.
+ */
+export function* resetEntireState() {
```

Comment:
all of the other function names for the reset actions mirror the constant names
and I think that would be a good precedent to follow here for consistency's sake.

so either:

- rename this function to `resetAllState()` 
- rename the constant to `RESET_ENTIRE_STATE`

depending on which one you think better describes the intent of the action,
I think my preference is maybe for the latter, but I'll leave that up to you